src/VoteProxy2.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.4.24;
+
+import 'ds-token/token.sol';
+import 'ds-chief/chief.sol';
+
+contract VoteProxy2 {
+    address public cold;
+    address public hot;
+    DSToken public GOV;
+    DSToken public IOU;
+    DSChief chief;
+
+    constructor(DSChief chief_, address cold_, address hot_) {
+        chief = chief_;
+        cold = cold_;
+        hot = hot_;
+        GOV = chief.GOV();
+        IOU = chief.IOU();
+        GOV.approve(chief, uint256(-1));
+        IOU.approve(chief, uint256(-1));
+    }
+
+    modifier onlyCold() { require(msg.sender == cold); _ }
+    modifier hotOrCold() { require(msg.sender == cold || msg.sender == hot); _ }
+
+    function vote(bytes32 slate)
+        hotOrCold
+    {
+        chief.vote(slate);
+    }
+    function lock(uint256 wad)
+        hotOrCold
+    {
+        chief.lock(wad);
+    }
+    function free(uint256 wad)
+        hotOrCold
+    {
+        chief.free(wad);
+    }
+    function release(uint256 wad)
+        onlyCold
```

Comment:
Why is this `onlyCold`, shouldn't the hot wallet also be allowed to release funds to the cold wallet? As it is now, the funds will be stuck in the proxy if the cold wallet is unable to make other calls than simple `transfer`s, which is exactly the scenario we are trying to solve. Or am I missing something?